src/components/VideoPlayerContexts/VideoPopoverMenuContext.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React, {useCallback, useContext, useMemo, useState} from 'react';
+import _ from 'underscore';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useLocalize from '@hooks/useLocalize';
+import fileDownload from '@libs/fileDownload';
+import * as Url from '@libs/Url';
+import CONST from '@src/CONST';
+import {usePlaybackContext} from './PlaybackContext';
+
+const VideoPopoverMenuContext = React.createContext(null);
+
+function VideoPopoverMenuContextProvider({children}) {
+    const {currentVideoPlayerRef} = usePlaybackContext();
+    const {translate} = useLocalize();
+    const [currentPlaybackSpeed, setCurrentPlaybackSpeed] = useState(CONST.VIDEO_PLAYER.PLAYBACK_SPEEDS[CONST.VIDEO_PLAYER.DEFAULT_PLAYBACK_SPEED_INDEX]);
```

Comment:
Not a blocker - I wonder if we really need a const for the default playback speed after all. `useState(1)` would make sense when you first read the code. We also don't seem to reuse the const anywhere else.